Sources/KituraNet/HTTP/HTTPServer.swift

```diff
@@ -114,6 +114,21 @@ public class HTTPServer : Server {
         return ctx.channel.pipeline.add(handler: webSocketHandlerFactory.handler(for: serverRequest))
     }
 
+    typealias ShouldUpgradeFunction = (HTTPRequestHead) -> HTTPHeaders?
+    typealias UpgradePipelineHandlerFunction = (Channel, HTTPRequestHead) -> EventLoopFuture<Void>
```

Comment:
Does it make sense to have these as `private` typealiases for now?

Reply:
Yes makes sense. Thanks :)